spec/factories/rooms.rb

```diff
@@ -1,7 +1,6 @@
 FactoryBot.define do
   factory :room do
     number { "42" }
-    floor { "E" }
```

Comment:
I would add this line again (see above, i opted against making the floor nullable)